### PR TITLE
Python3 & Iris2 compatibility check

### PR DESCRIPTION
**Pull request by kaedonkers**
Checked all four notebooks with Python3 + Iris2. All changes are to ensure compatibility. Only outstanding incompatibilities are four warnings, which will be detailed below.

### CODE REVIEW

**Comment by kaedonkers** [comment]
Outstanding warnings (discussions to be had about whether to change these):

Cell [28]
`IrisDeprecation: setting the 'Future' property 'cell_datetime_objects' is deprecated and will be removed in a future release. Please remove code that sets this property.`

Cell [60]
`UserWarning: Collapsing a non-contiguous coordinate. Metadata may not be fully descriptive for 'model_level_number'/'level_height'/'sigma'.`

Cells [67] & [68]
`UserWarning: Collapsing spatial coordinate 'latitude' without weighting`
`UserWarning: Collapsing a non-contiguous coordinate. Metadata may not be fully descriptive for 'latitude'.`

**Comment by kaedonkers** [comment]
Myself and @dkillick concluded that the UserWarnings can be left in place since they do not break the notebook and are more of a comment on the data than the code.

File: course_content/notebooks/iris_intro.ipynb [hunk 1]
```diff
@@ -757,7 +757,7 @@
    "source": [
     "### Time Constraints\n",
     "\n",
-    "It is common to want to build a constraint for time. This can be achieved by comparing cells containing datetimes. The functionality can be enabled globally within the session (and will be enabled by default in v2.0, the next release of Iris) with:"
+    "It is common to want to build a constraint for time. This can be achieved by comparing cells containing datetimes, which is available by default in Iris v2.0. If you are using Iris v1.x, please uncomment and run the line below to enable this functionality globally within the session:"
```
Comment (dkillick) [inline course_content/notebooks/iris_intro.ipynb]: "line" --> "code cell"

Maybe also remove the point versions from the Iris versions? (That is, v2.0 --> v2; v1.x --> v1) Or possibly, just v2.0 --> v2.x. But I'm not sold on this idea, so feel free to leave out this second bit.

File: course_content/notebooks/iris_intro.ipynb [hunk 2]
```diff
@@ -1002,9 +1002,9 @@
    },
    "outputs": [],
    "source": [
-    "print raw_cubes[0]\n",
-    "print\n",
-    "print raw_cubes[1]"
+    "print(raw_cubes[0])\n",
+    "print()\n",
```
Comment (dkillick) [inline course_content/notebooks/iris_intro.ipynb]: I just don't like an empty print call... [further down](https://github.com/SciTools/courses/pull/129/files#diff-6a5bf1866442f8f0072cf85e31079d02R1108) we use a line of dashes to perform the same job; we could replicate that here.

**Pull request merged**